### DISCUSSION TITLE
Webmin - host or lxc install

### DISCUSSION THREAD

**Discussion by holocronology** [tteck/Proxmox#3188] [General]
The notes on Webmin suggest installing it on an LXC, but if being used to manage the host node should it be installed on the host directly?

**Comment by MickLesk** [comment]
Where did you read that?
I have just looked at the source, it is installed directly in Proxmox, i.e. on its own shell.

**Reply by tteck** [reply]
![Screenshot 2024-06-13 9 50 51 AM](https://github.com/tteck/Proxmox/assets/21342670/f6a55deb-27f2-43ba-bd84-ff563111e8a4)

**Reply by holocronology** [reply]
Well, I think I introduced my own confusion when by "host" I meant NODE.

The language in the screen clip above implies that it is installed in the **LXC console** and thus each LXC.

**NODE shell** would imply that it is installed on the NODE.

Or is it both? My recollection is that the website previously indicated both, but I might be misremembering.

**Reply by Mellowlynx** [reply]
It's intended to be installed on an already deployed LXC, and not on a Proxmox node.

Proxmox had a web UI, and in a security/keep it simple way, you can better not install in on the host in my eyes.
So proceed with caution at your own risk.

**Reply by tismofied** [reply]
So the sole purpose is to install it in each LXC we would like to manage? I had it on the host itself for over a year  😨 
I guess I was lucky. I loved how it showed everything on proxmox and gave it a better GUI feel than the default proxmox one imo
